go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/styrainc/regal
 
-go 1.21
+go 1.20
```

Comment:
It would be good to have a comment here explaining our "strategy", which I take it would be to align with OPA, right?

Reply:
Honestly, aligning with OPA is worse -- OPA supports the supported golang versions and one unsupported version. I'm not entirely convinced that this is a good idea, so this is a compromise.

Reply:
Added a comment ✔️